src/attrs_strict/_type_validation.py

```diff
@@ -11,6 +13,11 @@
     UnionError,
 )
 
+
+class _StringAnnotationError(Exception):
+    pass
```

Comment:
Add a docstring explaining the use of this class to avoid needing to use ``pass``

Reply:
Why is this before other imports?